cql/ManageSpecialPopulation.cql

```diff
@@ -0,0 +1,644 @@
+/*
+  Library: Management of Abnormal Cervical Cancer Screening Results
+  Author: CMS Alliance to Modernize Healthcare, operated by THE MITRE Corporation.
+
+  (C) 2021 The MITRE Corporation. All Rights Reserved.
+  Approved for Public Release: 21-1556.
+  Distribution Unlimited.
+
+  Unless otherwise noted, this work is available under an Apache 2.0 license.
+  It was produced by the MITRE Corporation for the Division of Cancer Prevention
+  and Control, Centers for Disease Control and Prevention in accordance with the
+  Statement of Work, contract number 75FCMC18D0047, task order number 75D30120F09743.
+*/
+
+library ManageSpecialPopulation version '1.1.0'
+
+using FHIR version '4.0.1'
+
+include FHIRHelpers version '4.0.1' called FHIRHelpers
+include CDSConnectCommonsforFHIRv401 version '1.0.0' called C3F
+include DashboardLibrary version '1.0.0' called Dash
+include CollateManagementData version '1.1.0' called Collate
+include ManageRareAbnormality version '1.1.0' called Rare
+include CCSMCommonFunctions version '1.0.0' called CCF
+
+//------------------------------------------------------------------------------
+// CODE SYSTEMS, VALUE SETS, AND CODES
+//------------------------------------------------------------------------------
+
+codesystem "SNOMED-CT": 'http://snomed.info/sct'
+codesystem "ICD-9": 'http://hl7.org/fhir/sid/icd-9-cm'
+codesystem "ICD-10": 'http://hl7.org/fhir/sid/icd-10-cm'
+codesystem "NULL FLAVOR": 'http://terminology.hl7.org/CodeSystem/v3-NullFlavor'
+codesystem "LOCAL": 'http://OUR-PLACEHOLDER-URL.com'
+
+valueset "Premenopausal": 'http://cts.nlm.nih.gov/fhir/ValueSet/2.16.840.1.113762.1.4.1032.254' // AKA: Premenopausal
+valueset "All Results of High Risk HPV Test": 'http://cts.nlm.nih.gov/fhir/ValueSet/2.16.840.1.113762.1.4.1032.233'
+valueset "HSIL": 'http://cts.nlm.nih.gov/fhir/ValueSet/2.16.840.1.113762.1.4.1032.256'
+valueset "Normal Histology Finding": 'http://cts.nlm.nih.gov/fhir/ValueSet/2.16.840.1.113762.1.4.1032.262'
+
+code "ASC-H": '441088002' from "SNOMED-CT" display 'Atypical squamous cells on cervical Papanicolaou smear cannot exclude high grade squamous intraepithelial lesion (finding)'
+code "CIN1": '285836003' from "SNOMED-CT" display 'Cervical intraepithelial neoplasia grade 1 (disorder)'
```

Comment:
@dwwinters Many of these codes are now defined up to 4 times throughout the CDS. Could we make this DRYer by importing the codes from other libraries (i.e. from the Dashboard)? Or is the idea that we want them to be able to function somewhat independently from one another.

Reply:
My mistake.  Everyone should be referencing the codes defined in the dashboard.